Lesson4HomeWork.playground/Contents.swift

```diff
@@ -0,0 +1,21 @@
+import UIKit
+
+var arrayNumbers: [Int] = [-1, 2, 6, 12, 0, 5, 7, 8]
```

Comment:
We should create an array of tuples `(Int, String)`. Yes, an array can only contain elements of one type but `(Int, String)` is a type so you can create an array for it